config/mainCfg_TauTau_Legacy2018.cfg

```diff
@@ -1,42 +1,51 @@
 
-#variables = dau1_pt, dau2_pt, dau1_eta, dau2_eta, bjet1_pt, bjet2_pt, tauH_mass, tauH_SVFIT_mass, ditau_deltaR, tauH_pt, bH_pt, bH_mass, HH_pt, HH_mass_raw
-variables = tauH_pt
+# TOGETHER WITH THE TARGET VARIABLES YOU NEED ALWAYS PLOT LOWER LEVEL VARIABLES TO CHECK THAT EVERYTHING IS OK
+variables = dau1_pt, dau2_pt, dau1_eta, dau2_eta, ditau_deltaR, bjet1_pt, bjet1_bID_deepFlavor, bjet2_pt, bjet2_bID_deepFlavor, tauH_mass, tauH_SVFIT_mass, tauH_pt, bH_mass, bH_pt, HH_mass_raw, HH_pt, BDToutSM_kl_1
 
-selections = baseline, s1b1jresolved, s2b0jresolved, sboostedLL
+# SELECTIONS W/O MASS CUT
+selections = baseline, s1b1jresolved, s2b0jresolved, sboostedLL, s0b2j, VBFloose, VBFtight
 
-regions    = SR, SStight, SSrlx, OSinviso, SSinviso
+# SELECTIONS W/ MASS CUT
+#selections = baselineMcut, s1b1jresolvedMcut, s2b0jresolvedMcut, sboostedLLMcut, s0b2j, VBFlooseMcut, VBFtightMcut
+
+regions    = SR, SStight, OSrlx, SSrlx, OSinviso, SSinviso
 
 
 [configs]
 
 sampleCfg = config/sampleCfg_Legacy2018.cfg
 cutCfg    = config/selectionCfg_TauTau_Legacy2018.cfg
 
-[merge]
-TT        = TTfullyHad, TTfullyLep, TTsemiLep
 
-#WJets     = WJets_HT_0_100, WJets_HT_100_200, WJets_HT_200_400, WJets_HT_400_600, WJets_HT_600_800, WJets_HT_800_1200, WJets_HT_1200_2500, WJets_HT_2500_Inf
-W     = WJets_HT_0_100, WJets_HT_100_200, WJets_HT_200_400, WJets_HT_400_600, WJets_HT_600_800, WJets_HT_800_1200, WJets_HT_1200_2500, WJets_HT_2500_Inf
+[merge]
+# HERE THE MERGING OF THE BKGs IS DONE SO THAT WE CAN COMPUTE DURING THE PLOTTING ALSO THE YIELDS OF THE SIGLE PROCESSES INVOLVED
+# IN ORDER TO HAVE THE SAME FINAL PLOTS AS FOR THE OTHER YEARS, WHEN PLOTTING THE MERGING IS FURTHER CARRIED OUT AS FOLLOWS:
+# others = EWK + singleT + VH + singleT + VH + ttH + doubleTsingleV + doubleTVV + doubleV + tripleV + VBFHTauTau + ggHTauTau
 
-#other	  = TWtop, TWantitop, singleTop_top, singleTop_antitop, EWKWMinus2Jets_WToLNu, EWKWPlus2Jets_WToLNu, EWKZ2Jets_ZToLL, EWKZ2Jets_ZToNuNu, WW, WZ, ZZ, ZH_HBB_ZLL, ZH_HTauTau, ggHTauTau, VBFHTauTau, WplusHTauTau, WminusHTauTau, ttHJetTononBB, ttHJetToBB, WWW, WWZ, WZZ, TTWJetsToLNu, TTWJetsToQQ, TTWW
-EWK     = EWKWMinus2Jets_WToLNu, EWKWPlus2Jets_WToLNu, EWKZ2Jets_ZToLL, EWKZ2Jets_ZToNuNu
+TT        = TTfullyHad, TTfullyLep, TTsemiLep
+WJets     = WJets_HT_0_100, WJets_HT_100_200, WJets_HT_200_400, WJets_HT_400_600, WJets_HT_600_800, WJets_HT_800_1200, WJets_HT_1200_2500, WJets_HT_2500_Inf
```

Comment:
don't we have two different samples: `WJets_HT_0_70` and `WJets_HT_70_100`?

Reply:
Yes, but from the SKIM that @camendola produced we only have WJets_HT_0_100 so I thought for some reason it had been taken the decision to merge them in the 2018 dataset.

@camendola what do you say?

Reply:
Not in 2018, binned datasets start from HT-100